pkg/apis/hobbyfarm.io/v1/types.go

```diff
@@ -222,6 +222,36 @@ type VirtualMachineSetList struct {
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
+type Course struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              CourseSpec `json:"spec"`
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type CourseList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata"`
+	Items           []Course `json:"spec"`
```

Comment:
@hall @Oats87 
This is the line causing the cache sync issues for Courses. 
`json:"items"` instead of `json:"spec"`

Reply:
:eagle: :eye:! Fixed -- thanks, @ebauman.